algosdk/future/transaction.py

```diff
@@ -3181,9 +3184,6 @@ def wait_for_confirmation(
     last_round = algod_client.status()["last-round"]
     current_round = last_round + 1
 
-    if wait_rounds == 0:
-        wait_rounds = 1000
-
```

Comment:
@jasonpaulos Do you consider the following proposal a worthwhile improvement that retains backwards compatibility?

* Restore the highlighted, deleted lines so that callers supplying `wait_rounds = 0` retain the existing behavior + leave rest of PR as is.
* Optionally, add a brief inline comment explaining the behavior.

Otherwise, I agree the PR proposes a backwards incompatible change.

Reply:
So are you suggesting to _only_ change the default value of the `wait_rounds` parameter from 0 to 1000? That seems fine to me.

Independently of that change, I agree we should document _why_ the special case for 0 must exist, and explain this in the docstring as well.

Reply:
> you suggesting to only change the default value of the wait_rounds parameter from 0 to 1000? 

@jasonpaulos Correct.

@efe Provided you're on-board too, can you make the change + add a docstring remark explaining the significance of the 0 value?